scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -146,7 +151,46 @@
                       {% endfor %}
                   ]
               };
-      igv.createBrowser(div, options);
+      igv.createBrowser(div, options)
+      .then(function (browser) {
+          browser.on('trackclick', function (track, popoverData) {
+
+              var markup = "<table class=\"igv-popover-table\">";
+
+              // Don't show a pop-over when there's no data.
+              if (!popoverData || !popoverData.length) {
+                  return false;
+              }
+
+              popoverData.forEach(function (nameValue) {
+
+              if (nameValue.name) {
+
+                  var value = nameValue.value;
+                  for (var key in outLinks) {
+                    if (nameValue.value.toString().startsWith(key) && nameValue.name.toLowerCase() === 'name') {
+                      value = '<a href="' + outLinks[key] + value + '" target="_blank" rel="noopener noreferrer">' + nameValue.value + '</a>';
+                    }
+                  }
+
+                  markup += "<tr><td class=\"igv-popover-td\">"
```

Comment:
Since we are not really using a second column, sth like an `ul` with `li` items would make more sense to me, but that is just style. Maybe there is even some CSS thing going that I'm missing?

Reply:
I wanted to stick to the code suggested by igv.js --> https://igv.org/web/release/2.7.4/examples/events/custom-track-popover.html. Regarding the css I'm not sure. 

I can remove the class (it looks OK already I think) or include the missing code in a css code I guess. I'll fix